package.json

```diff
@@ -24,85 +24,85 @@
   "author": "Mozilla (https://mozilla.org/)",
   "license": "MPL-2.0",
   "dependencies": {
-    "bluebird": "2.10.1",
-    "body-parser": "1.14.1",
-    "bower": "1.7.1",
+    "bluebird": "3.3.5",
+    "body-parser": "1.15.0",
+    "bower": "1.7.9",
     "connect-cachify": "0.0.17",
     "connect-fonts-clearsans": "0.0.1",
     "connect-fonts-firasans": "0.0.4",
-    "consolidate": "0.13.1",
-    "convict": "1.0.2",
-    "cookie-parser": "1.4.0",
+    "consolidate": "0.14.0",
+    "convict": "1.2.0",
+    "cookie-parser": "1.4.1",
     "cors": "2.7.1",
-    "express": "4.13.3",
+    "express": "4.13.4",
     "express-able": "0.4.4",
     "extend": "3.0.0",
     "grunt": "0.4.5",
```

Comment:
not updating this to 1.0 for now due to CI blockage,  but picking up the latest plugins anyway